arenadata/Dockerfile.ubuntu

```
@@ -31,25 +32,32 @@ RUN set -eux; \
 
 WORKDIR /home/gpadmin
 
-FROM base as build
+FROM base AS build
 
 COPY . gpdb_src
 
 RUN mkdir /home/gpadmin/bin_gpdb
 
 ENV TARGET_OS=ubuntu
+ENV PYTHON=/usr/bin/python2
```

why not
```suggestion
export PYTHON=...
```
in RUN section?